File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Ocsa/DOC4003U00GrdDOC4003Info.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Ocsa
{
    [Serializable]
    public class DOC4003U00GrdDOC4003Info
    {
        private String _sysDate;
        private String _sysId;
        private String _updDate;
        private String _updId;
        private String _hospCode;
        private String _pkdoc4003;
        private String _seq;
        private String _createDate;
        private String _toHospitalInfo;
        private String _toSinryouka;
        private String _toSinryouka2;
        private String _toDoctor;
        private String _toDoctor2;
        private String _doctor;
        private String _gwa;
        private String _suname;
        private String _sex;
        private String _zip;
        private String _aAddress;
        private String _tel;
        private String _birth;
        private String _job;
        private String _disease;
        private String _checkupOpinion;
        private String _prescription;
        private String _bunho;
        private String _zipCode;
        private String _bAddress;
        private String _fax;
        private String _yoyangName;
        private String _rowState;

        public String SysDate
        {
            get { return this._sysDate; }
            set { this._sysDate = value; }
        }

        public String SysId
        {
            get { return this._sysId; }
            set { this._sysId = value; }
        }

        public String UpdDate
        {
            get { return this._updDate; }
            set { this._updDate = value; }
        }

        public String UpdId
        {
            get { return this._updId; }
            set { this._updId = value; }
        }

        public String HospCode
        {
            get { return this._hospCode; }
            set { this._hospCode = value; }
        }

        public String Pkdoc4003
        {
            get { return this._pkdoc4003; }
            set { this._pkdoc4003 = value; }
        }

        public String Seq
        {
            get { return this._seq; }
            set { this._seq = value; }
        }

        public String CreateDate
        {
            get { return this._createDate; }
            set { this._createDate = value; }
        }

        public String ToHospitalInfo
        {
            get { return this._toHospitalInfo; }
            set { this._toHospitalInfo = value; }
        }

        public String ToSinryouka
        {
            get { return this._toSinryouka; }
            set { this._toSinryouka = value; }
        }

        public String ToSinryouka2
        {
            get { return this._toSinryouka2; }
            set { this._toSinryouka2 = value; }
        }

        public String ToDoctor
        {
            get { return this._toDoctor; }
            set { this._toDoctor = value; }
        }

        public String ToDoctor2
        {
            get { return this._toDoctor2; }
            set { this._toDoctor2 = value; }
        }

        public String Doctor
        {
            get { return this._doctor; }
            set { this._doctor = value; }
        }

        public String Gwa
        {
            get { return this._gwa; }
            set { this._gwa = value; }
        }

        public String Suname
        {
            get { return this._suname; }
            set { this._suname = value; }
        }

        public String Sex
        {
            get { return this._sex; }
            set { this._sex = value; }
        }

        public String Zip
        {
            get { return this._zip; }
            set { this._zip = value; }
        }

        public String AAddress
        {
            get { return this._aAddress; }
            set { this._aAddress = value; }
        }

        public String Tel
        {
            get { return this._tel; }
            set { this._tel = value; }
        }

        public String Birth
        {
            get { return this._birth; }
            set { this._birth = value; }
        }

        public String Job
        {
            get { return this._job; }
            set { this._job = value; }
        }

        public String Disease
        {
            get { return this._disease; }
            set { this._disease = value; }
        }

        public String CheckupOpinion
        {
            get { return this._checkupOpinion; }
            set { this._checkupOpinion = value; }
        }

        public String Prescription
        {
            get { return this._prescription; }
            set { this._prescription = value; }
        }

        public String Bunho
        {
            get { return this._bunho; }
            set { this._bunho = value; }
        }

        public String ZipCode
        {
            get { return this._zipCode; }
            set { this._zipCode = value; }
        }

        public String BAddress
        {
            get { return this._bAddress; }
            set { this._bAddress = value; }
        }

        public String Fax
        {
            get { return this._fax; }
            set { this._fax = value; }
        }

        public String YoyangName
        {
            get { return this._yoyangName; }
            set { this._yoyangName = value; }
        }

        public String RowState
        {
            get { return this._rowState; }
            set { this._rowState = value; }
        }

        public DOC4003U00GrdDOC4003Info() { }

        public DOC4003U00GrdDOC4003Info(String sysDate, String sysId, String updDate, String updId, String hospCode, String pkdoc4003, String seq, String createDate, String toHospitalInfo, String toSinryouka, String toSinryouka2, String toDoctor, String toDoctor2, String doctor, String gwa, String suname, String sex, String zip, String aAddress, String tel, String birth, String job, String disease, String checkupOpinion, String prescription, String bunho, String zipCode, String bAddress, String fax, String yoyangName, String rowState)
        {
            this._sysDate = sysDate;
            this._sysId = sysId;
            this._updDate = updDate;
            this._updId = updId;
            this._hospCode = hospCode;
            this._pkdoc4003 = pkdoc4003;
            this._seq = seq;
            this._createDate = createDate;
            this._toHospitalInfo = toHospitalInfo;
            this._toSinryouka = toSinryouka;
            this._toSinryouka2 = toSinryouka2;
            this._toDoctor = toDoctor;
            this._toDoctor2 = toDoctor2;
            this._doctor = doctor;
            this._gwa = gwa;
            this._suname = suname;
            this._sex = sex;
            this._zip = zip;
            this._aAddress = aAddress;
            this._tel = tel;
            this._birth = birth;
            this._job = job;
            this._disease = disease;
            this._checkupOpinion = checkupOpinion;
            this._prescription = prescription;
            this._bunho = bunho;
            this._zipCode = zipCode;
            this._bAddress = bAddress;
            this._fax = fax;
            this._yoyangName = yoyangName;
            this._rowState = rowState;
        }

    }
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/ocsa/DOC4003U00GrdDOC4003Info.java
package nta.med.data.model.ihis.ocsa;

import java.io.Serializable;
import java.sql.Timestamp;

public class DOC4003U00GrdDOC4003Info implements Serializable {
	private Timestamp sysDate;
	private String sysId;
	private String updDate;
	private String updId;
	private String hospCode;
	private Double pkdoc4003;
	private Double seq;
	private String createDate;
	private String toHospitalInfo;
	private String toSinryouka;
	private String toSinryouka2;
	private String toDoctor;
	private String toDoctor2;
	private String doctor;
	private String gwa;
	private String suname;
	private String sex;
	private String zip;
	private String aAddress;
	private String tel;
	private String birth;
	private String job;
	private String disease;
	private String checkupOpinion;
	private String prescription;
	private String bunho;
	private String zipCode;
	private String bAddress;
	private String fax;
	private String yoyangName;
	private String rowState;

	public DOC4003U00GrdDOC4003Info(Timestamp sysDate, String sysId, String updDate, String updId, String hospCode,
			Double pkdoc4003, Double seq, String createDate, String toHospitalInfo, String toSinryouka,
			String toSinryouka2, String toDoctor, String toDoctor2, String doctor, String gwa, String suname,
			String sex, String zip, String aAddress, String tel, String birth, String job, String disease,
			String checkupOpinion, String prescription, String bunho, String zipCode, String bAddress, String fax,
			String yoyangName) {
		super();
		this.sysDate = sysDate;
		this.sysId = sysId;
		this.updDate = updDate;
		this.updId = updId;
		this.hospCode = hospCode;
		this.pkdoc4003 = pkdoc4003;
		this.seq = seq;
		this.createDate = createDate;
		this.toHospitalInfo = toHospitalInfo;
		this.toSinryouka = toSinryouka;
		this.toSinryouka2 = toSinryouka2;
		this.toDoctor = toDoctor;
		this.toDoctor2 = toDoctor2;
		this.doctor = doctor;
		this.gwa = gwa;
		this.suname = suname;
		this.sex = sex;
		this.zip = zip;
		this.aAddress = aAddress;
		this.tel = tel;
		this.birth = birth;
		this.job = job;
		this.disease = disease;
		this.checkupOpinion = checkupOpinion;
		this.prescription = prescription;
		this.bunho = bunho;
		this.zipCode = zipCode;
		this.bAddress = bAddress;
		this.fax = fax;
		this.yoyangName = yoyangName;
	}

	public Timestamp getSysDate() {
		return sysDate;
	}

	public void setSysDate(Timestamp sysDate) {
		this.sysDate = sysDate;
	}

	public String getSysId() {
		return sysId;
	}

	public void setSysId(String sysId) {
		this.sysId = sysId;
	}

	public String getUpdDate() {
		return updDate;
	}

	public void setUpdDate(String updDate) {
		this.updDate = updDate;
	}

	public String getUpdId() {
		return updId;
	}

	public void setUpdId(String updId) {
		this.updId = updId;
	}

	public String getHospCode() {
		return hospCode;
	}

	public void setHospCode(String hospCode) {
		this.hospCode = hospCode;
	}

	public Double getPkdoc4003() {
		return pkdoc4003;
	}

	public void setPkdoc4003(Double pkdoc4003) {
		this.pkdoc4003 = pkdoc4003;
	}

	public Double getSeq() {
		return seq;
	}

	public void setSeq(Double seq) {
		this.seq = seq;
	}

	public String getCreateDate() {
		return createDate;
	}

	public void setCreateDate(String createDate) {
		this.createDate = createDate;
	}

	public String getToHospitalInfo() {
		return toHospitalInfo;
	}

	public void setToHospitalInfo(String toHospitalInfo) {
		this.toHospitalInfo = toHospitalInfo;
	}

	public String getToSinryouka() {
		return toSinryouka;
	}

	public void setToSinryouka(String toSinryouka) {
		this.toSinryouka = toSinryouka;
	}

	public String getToSinryouka2() {
		return toSinryouka2;
	}

	public void setToSinryouka2(String toSinryouka2) {
		this.toSinryouka2 = toSinryouka2;
	}

	public String getToDoctor() {
		return toDoctor;
	}

	public void setToDoctor(String toDoctor) {
		this.toDoctor = toDoctor;
	}

	public String getToDoctor2() {
		return toDoctor2;
	}

	public void setToDoctor2(String toDoctor2) {
		this.toDoctor2 = toDoctor2;
	}

	public String getDoctor() {
		return doctor;
	}

	public void setDoctor(String doctor) {
		this.doctor = doctor;
	}

	public String getGwa() {
		return gwa;
	}

	public void setGwa(String gwa) {
		this.gwa = gwa;
	}

	public String getSuname() {
		return suname;
	}

	public void setSuname(String suname) {
		this.suname = suname;
	}

	public String getSex() {
		return sex;
	}

	public void setSex(String sex) {
		this.sex = sex;
	}

	public String getZip() {
		return zip;
	}

	public void setZip(String zip) {
		this.zip = zip;
	}

	public String getaAddress() {
		return aAddress;
	}

	public void setaAddress(String aAddress) {
		this.aAddress = aAddress;
	}

	public String getTel() {
		return tel;
	}

	public void setTel(String tel) {
		this.tel = tel;
	}

	public String getBirth() {
		return birth;
	}

	public void setBirth(String birth) {
		this.birth = birth;
	}

	public String getJob() {
		return job;
	}

	public void setJob(String job) {
		this.job = job;
	}

	public String getDisease() {
		return disease;
	}

	public void setDisease(String disease) {
		this.disease = disease;
	}

	public String getCheckupOpinion() {
		return checkupOpinion;
	}

	public void setCheckupOpinion(String checkupOpinion) {
		this.checkupOpinion = checkupOpinion;
	}

	public String getPrescription() {
		return prescription;
	}

	public void setPrescription(String prescription) {
		this.prescription = prescription;
	}

	public String getBunho() {
		return bunho;
	}

	public void setBunho(String bunho) {
		this.bunho = bunho;
	}

	public String getZipCode() {
		return zipCode;
	}

	public void setZipCode(String zipCode) {
		this.zipCode = zipCode;
	}

	public String getbAddress() {
		return bAddress;
	}

	public void setbAddress(String bAddress) {
		this.bAddress = bAddress;
	}

	public String getFax() {
		return fax;
	}

	public void setFax(String fax) {
		this.fax = fax;
	}

	public String getYoyangName() {
		return yoyangName;
	}

	public void setYoyangName(String yoyangName) {
		this.yoyangName = yoyangName;
	}

	public String getRowState() {
		return rowState;
	}

	public void setRowState(String rowState) {
		this.rowState = rowState;
	}

}
